django_celery_beat/models.py

```diff
@@ -616,7 +626,7 @@ def schedule(self):
         if self.interval:
             return self.interval.schedule
         if self.crontab:
-            return self.crontab.schedule
+            return self.crontab
```

Comment:
Some unit tests are failed because of this change.
Can you elaborate more on why we need this?

Reply:
I recently closed PR #621 because I realized the changes are already covered in yours.
Hope this can give you an idea.

Reply:
I can merge part of your PR first @khanh96le

Reply:
Sure @auvipy. I saw that you already reopened it

Reply:
I like the change you made in #621, @khanh96le! That's an improved list view!